src/ui/public/courier/fetch/request/_error_handler.js

```diff
@@ -14,7 +14,7 @@ define(function (require) {
       });
 
       if (!myHandlers.length) {
-        notify.fatal(new Error('unhandled error ' + (error.stack || error.message)));
+        notify.fatal(new Error(`unhandled courier request error: ${ notify.describeError(error) }`));
```

Comment:
Right, that was intentional. The stack trace to this location is always the same. There isn't any value in the non-async stack trace